docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,6 +2,61 @@
 
 These workflows for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
+_Last updated 27-Mar-2024_
+### Preferred Instructions
+
+1. Open the `src/ontology/imports/manual_seed.txt`
+2. Add the IRI of the term(s) you want to add to this document, and save
+    - you can add any IRI, e.g. gene, NCBITaxon,...
+3. In the Terminal, run: `export "MEMORY_GB=15"`
+4. Then run import: `sh run.sh make refresh-merged` 
+    - All the imports will be updated, which means that you might see changes in your GitHub diff in the following files
+        - `src/ontology/imports/*_terms.txt`
+        - `src/ontology/imports/merged_import.owl`
+    - The terms added in the "manual_seed.txt" will be added to the appropriate import (e.g human genes will be added to hgnc_terms.txt; NCBITaxon will be added to ncbitaxon_terms.txt), and removed from the "manual_seed.txt" after the import refresh has run. 
+5. Edit in Protege, the new terms should be available for logical definitions.
```

Comment:
Does `NORM` need to be run as part of this process?

Reply:
Ok, I guess if step 5 "Edit in Protege" is always done after running the import then the following Save in Protege of the updates to add the logical definitions will take care of any issues. I wasn't sure if someone might refresh the imports, but not edit in Protege immediately afterwards.